src/Stripe.net/Services/FileLinks/StripeFileLinkListOptions.cs

```diff
@@ -0,0 +1,16 @@
+namespace Stripe
+{
+    using Newtonsoft.Json;
+
+    public class StripeFileLinkListOptions : StripeListOptions
```

Comment:
Minor but there's a class called `StripeListOptionsWithCreated` that should either be used everywhere we have created or canned.